apps/website/src/pages/docs/inputgrid.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Input grid
+description: Used to display inputs with label and/or status message.
+layout: ./_layout.astro
+propsPath: '@itwin/itwinui-react/esm/core/InputGrid/InputGrid.d.ts'
+thumbnail: #TODO
+group: utilities
+---
+
+import PropsTable from '~/components/PropsTable.astro';
+import LiveExample from '~/components/LiveExample.astro';
+import * as AllExamples from 'examples';
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='InputGrid.main.tsx' truncate={false}>
+  <AllExamples.InputGridExample client:load />
+</LiveExample>
+
+InputGrid is a layout component that allows to group label, input and status message.
+
+You can add any input (eg. Select, ComboBox or InputWithDecorations).
+
```

Comment:
needs more examples? at least one for inline label

Reply:
Added